backend/inspirehep/records/api/literature.py

```diff
@@ -185,7 +179,7 @@ def add_files(self):
             builder.add_document(**document)
             keys.append(document["key"])
             if "documents" in builder.record:
```

Comment:
This should be outside for loop otherwise you update it every loop which is unnecessary.

Reply:
oups missed that